pkg/k8ssecret/secret.go

```diff
@@ -165,3 +164,13 @@ func GenerateK8SSecret(
 
 	return secret, nil
 }
+
+func SecretLength(secret *corev1.Secret) int {
```

Comment:
Maybe we can add a small test for this.

Reply:
we can. I might :)

The thing with this here: I honestly don't know what it counts. I don't know what the expected behaviour is exactly. All I am expecting that if secret A has more lines in the Data section then secret B, then the number returned is higher. I might add a test for that.

Reply:
A simple unit test would work here. It seems secret data holds `byte` by `string`. The test would be:

```go
func TestSecretLength(t *testing.T) {
	secret := corev1.Secret{
		Data: map[string][]byte{
			"foo": []byte("bar"),
		},
	}

	expectedSecret := corev1.Secret{
		Data: map[string][]byte{
			"foo": []byte("bar1"),
		},
	}
	secretLength := SecretLength(&secret)
	expectedSecretLength := SecretLength(&expectedSecret)

	if expectedSecretLength <= secretLength {
		t.Errorf("secrets should not be equal")
	}
}
```

Reply:
@omegion added!